content/d3/concepts/selection/terms/selectAll/selectAll.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'selectAll()'
+Description: 'A brief definition and examples of using selectAll()'
```

Comment:
```suggestion
Title: '.selectAll()'
Description: 'A brief definition and examples of using selectAll()'
```
Change the description and add the actual description of the method in one line.